REVIEWING.md

```diff
@@ -1,27 +1,49 @@
 # Reviewing and Merging Pull Requests
 
-This document is a guideline for how the project's maintainers will review 
+As a community we believe in the value of code reviews for all contributions.
+Code reviews increase both the quality and readability of our code base, which
+in turn produces high quality software.
+
+This document provides guidelines for how the project's maintainers review
 issues and merge pull requests (PRs).
 
-## Project Maintainers
+- [Pull requests welcome](#pull-requests-welcome)
+- [Code reviewers](#code-reviewers)
+- [Reviewing changes](#reviewing-changes)
+  - [Holds](#holds)
+- [Project maintainers](#project-maintainers)
+- [Merging PRs](#merging-prs)
+  - [Merge hours](#merge-hours)
 
-Reviewing of issues and PRs is done by the projects maintainers. The current
-list of maintainers is kept in the [OWNERS](OWNERS) file at the root of
-each repository.
+## Pull requests welcome
 
-Like many opensource projects, becoming a maintainer is based on contributions
-to the project. Active contributors to the project will eventually get noticed
-by the maintainers, and one of the existing maintainers will nominate the
-contributor to become a maintainer.  A 'yes' vote from >75% of
-the existing maintainers is required for approval.
+First and foremost: as a potential contributor, your changes and ideas are
+welcome at any hour of the day or night, weekdays, weekends, and holidays.
+Please do not ever hesitate to ask a question or send a PR.
 
-Removing a maintainer requires a 'yes' vote from >75% of the exsiting
-maintainers. Note that inactive maintainers might periodically be removed
-simply to keep the membership list accurate.
+### Code reviewers
+
+The code review process can introduce latency for contributors
+and additional work for reviewers that can frustrate both parties.
+Consequently, as a community we expect that all active participants in the
+community will also be active reviewers. We ask that active contributors to
+the project participate in the code review process in areas where that
+contributor has expertise.
 
-## Reviewing
+Active contributors are considered to be anyone who meets any of the following
```

Comment:
I would remove this section. I'm not sure there's value in defining "active" since it will really vary based on what a person does. It also opens the door for people to try to game the system. E.g. I submit a ton of PRs, all valid, but they're typos in comments.  Yes we're seen this.  I'm "active" but I'm not really as "valuable" to the community as someone who submit real code changes.  I think by removing this section and just using the term "active contributor" down below on line 95 we're ok - people will know what it means and leave us the freedom to be subjective about people's contributions.